ui/packages/consul-peerings/app/templates/dc/peers/edit/addresses.hbs

```diff
@@ -1,7 +1,40 @@
-<Route
-  @name={{routeName}}
-as |route|>
-  <Consul::Peer::Address::List
-    @items={{route.model.items}}
-  />
-</Route>
+<Route @name={{routeName}} as |route|>
+  {{#if (gt route.model.items.length 0)}}
+    <Consul::Peer::Address::List @items={{route.model.items}} />
+  {{else}}
+    <EmptyState @login={{route.model.app.login.open}}>
+      <BlockSlot @name="header">
+        <h2>
+          No server addresses.
+        </h2>
+      </BlockSlot>
+      <BlockSlot @name="body">
+        <div>
+          There don't seem to be any server addresses for this peer.
+        </div>
+      </BlockSlot>
+      <BlockSlot @name="actions">
+        <li class="docs-link">
+          <a
+            href="{{env 'CONSUL_DOCS_URL'}}/connect/cluster-peering"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            Documentation on Peers
+          </a>
+        </li>
+        <li class="learn-link">
+          <a
+            href="{{env
+              'CONSUL_DOCS_URL'
+            }}/connect/cluster-peering/create-manage-peering"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            Take the tutorial
```

Comment:
It would be nice to use the translations file for this copy

Reply:
Done